MethodsHomework/MethodsHomework.cs

```diff
@@ -106,66 +106,184 @@ public void UsingMethods()
-            bool doesContaintFox = /*Answer here*/;
+            bool doesContaintFox = sentence.Contains("fox");
             Grading.ContainsFox(doesContaintFox);
 
             // Find a method to replace the word quick, with the word slow
             // you must capture the result of the found method;
-            sentence.Replace("quick", "slow");
-            string newSentence = /*Answer here*/;
+            
+            string newSentence = sentence.Replace("quick", "slow"); 
             Grading.ReplacedTheWord(newSentence);
 
             // Find a method to Convert this word to lowercase;
             string word = "UPPER";
             // you must capture the result of the found method;
-            string lowered = /*Answer here*/;
+            string lowered = word.ToLower();
             Grading.StringWasLowercased(lowered);
 
             // Find a method to Convert this word to lowercase;
             string needsSplit = "I,Need,These,Separate";
             // you must capture the result of the found method;
-            string[] split = /*Answer here*/;
+            string[] split = needsSplit.Split(',' , 4);
             Grading.StringWasSplit(split);
 
 
             // I hope by now you see the power of what the types already do for you.
         }
+
+        
     }
 
     public class YourAnswers
     {
         //READ ME
         // You write all your methods here.
+        public static void DoNothing()
+        {
+
+        }
+
+        public static int ReturnIntWithValue25()
+        {
+            return 25;
+        }
+
+        public static int AddTwoNumbers(int num1, int num2)
+        {
+            int sum = num1 + num2;
+            return sum;
+        }
+
+        public static int IncrementMe(int valueToIncrement)
+        {
+            int increment = valueToIncrement + 1;
+            return increment;
+        }
+
+        public static int DecrementMe(int valueToDecrement)
+        {
+            int Decrement = valueToDecrement - 1;
+            return Decrement;
+        }
+
+        public static int CastDoubleToInt(double valueToBeCasted)
+        {
+            int casted = (int)valueToBeCasted;
+            return casted;
+        }
+
+
+        public static double ConvertIntToDouble (int valueToBeConverted)
+        {
+            double converted = Convert.ToDouble(valueToBeConverted);
+            return converted;
+        }
+
+        public static string CastMeIntoAString( double valueToBeCastedIntoString)
+        {
+            string casted = Convert.ToString(valueToBeCastedIntoString);
+            return casted;
+        }
+
+        public static double ParseMeIntoADouble(string value)
+        {
+            double parseValue = Double.Parse(value);
+            return parseValue;
+        }
+
+        public static int ReturnMaxIntValue()
+        {
+            return int.MaxValue;
+        }
+
+        public static bool CheckIfIAmTrue(bool whatIf)
```

Comment:
This method returns the expected result when I call `YourAnswers.CheckIfIAmTrue(false)`. But what happens if I call `YourAnswers.CheckIfIAmTrue(true)`?